app/scripts/modules/tencentcloud/src/vpc/VpcReader.ts

```diff
@@ -0,0 +1,39 @@
+import { IPromise } from 'angular';
+
+import { IVpc, NetworkReader } from '@spinnaker/core';
+export interface ITencentcloudVpc extends IVpc {
+  isDefault?: boolean;
+}
+export class VpcReader {
```

Comment:
`VPC` is an [Amazon Web Services term](https://aws.amazon.com/vpc/).  I see that it is also a [Tencent Cloud term](https://intl.cloud.tencent.com/product/vpc).  Are they the same thing?

Reply:
yes.